examples/Huggingface_Transformers/prepare_mar_from_hf.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+HF_REPO_URL=""
```

Comment:
need to make sure to whole example is tested and actually runs in `serve/test/pytest`